### PR TITLE
Let multio.init() accept modules too

### PR DESCRIPTION
**Pull request by pmav99**

### CODE REVIEW

**Comment by raylu** [comment]
thanks for fixing pylint!

File: multio/tests/__init__.py [hunk 1]
```diff
@@ -0,0 +1 @@
+# -*- coding: utf-8 -*-
```
Comment (raylu) [inline multio/tests/__init__.py]: these PEP 263 source code encodings aren't needed in python 3

Comment (pmav99) [reply multio/tests/__init__.py]: I removed them.
TBH, I was tempted to use "keep @raylu happy" as the commit message! :P :D

Comment (theelous3) [reply multio/tests/__init__.py]: About 50% of my commits are for just that reason ^

**Pull request merged**